demos/using-dev-build/package.json

```diff
@@ -5,18 +5,27 @@
   "scripts": {
     "wdio": "wdio run ./wdio.conf.ts",
     "test": "./test.js",
-    "format": "prettier --write --plugin-search-dir=. ."
+    "format": "prettier --write --plugin-search-dir=. .",
+    "build": "webpack",
+    "start": "webpack serve",
+    "proxy": "./test.js --no-run"
   },
   "devDependencies": {
+    "@dfinity/auth-client": "^0.9.3",
     "@wdio/cli": "^7.16.13",
     "@wdio/local-runner": "^7.16.13",
     "@wdio/mocha-framework": "^7.16.13",
     "@wdio/spec-reporter": "^7.16.13",
     "chromedriver": "^100.0.0",
     "prettier": "^2.5.0",
     "proxy": "git+https://github.com/nmattia/dfx-proxy",
+    "html-webpack-plugin": "5.3.2",
+    "copy-webpack-plugin": "^9.0.1",
+    "process": "0.11.10",
     "ts-node": "^10.4.0",
     "typescript": "^4.5.4",
+    "webpack": "5.45.1",
+    "webpack-cli": "4.7.2",
```

Comment:
I had to use `4.9.1`

https://github.com/webpack/webpack-cli/issues/2990#issuecomment-1003334175